src/Introduction.md

```diff
@@ -89,31 +89,31 @@ Clam AntiVirus is highly cross-platform. The development team cannot test every
 
 - GNU/Linux
   - Alpine
-    - 3.17 (x86_64)
+    - 3.21 (x86_64, arm64)
   - Ubuntu
-    - 18.04 (x86_64, i386)
-    - 20.04 (x86_64)
+    - 24.04 (x86_64, arm64)
+    - 22.04 (x86_64, arm64)
   - Debian
-    - 10 (x86_64, i386)
-    - 11 (x86_64, i386)
-  - CentOS
-    - 7 (x86_64, i386)
+    - 12 (x86_64, arm64)
+    - 11 (x86_64, arm64)
+  - AlmaLinux
+    - 8.10 (x86_64, arm64)
   - Fedora
-    - 31 (x86_64)
-    - 33 (x86_64)
+    - 41 (x86_64, arm64)
+    - 40 (x86_64, arm64)
   - openSUSE
-    - 15 Leap (x86_64)
+    - 15 Leap (x86_64, arm64)
 - UNIX
   - FreeBSD
-    - 12 (x86_64)
+    - 14 (x86_64)
     - 13 (x86_64)
   - macOS
-    - 10.13 High Sierra (Intel x86_64)
-    - 10.15 Catalina (Intel x86_64)
-    - 11.5 Big Sur (Intel x86_64, arm64 Apple M1)
+    - 15.3 Sequoia (x86_64, arm64)
+    - 14.7 Sonoma (x86_64, arm64)
+    - 13.7 Ventura (x86_64, arm64)
 - Windows
-  - 7 (x86_64, i386)
-  - 10 (x86_64, i386)
+  - 11 (x86_64, arm64)
+  - 10 (x86_64, arm64)
```

Comment:
We still build/publish/test packages for Win 10 i386. We don't technically test arm64 for Windows 10 (just 11) though I don't know of any reason it wouldn't work.